src/intl/messages/oc.json

```diff
@@ -1,6 +1,10 @@
 {
   "text.title": "Consentiment als cookies",
   "text.description": "En clicant Acceptar, consentètz a l'utilizacion de cookies per nòstre site Web per vos ofrir l'experiéncia mai pertinenta en memorizant vòstras preferéncias a prepaus dels cookies.",
+  "text.customizeTitle": "",
   "button.accept": "Acceptar",
-  "button.decline": "Refusar"
+  "button.acceptAll": "",
+  "button.declineAll": "",
+  "button.customize": "",
+  "button.back": ""
```

Comment:
hi @ensag-dev, I would like to know if you can help me translate these lines in oc please? thank you!